DM-3290 - LWM2M ignore empty values flag (#1737)
SoftwareAG/c8y-docs: content/protocol-integration/lwm2m-bundle/device-protocols.md

![Custom action example](/images/device-protocols/lwm2m/lwm2m-custom-action-connectivity-signal.png)

> ##### Ignore empty values {#ignore-empty-values}

Reviewer: this is not only for custom action so it can be outside of custom actions section.

> When this option is selected and device sends empty data, no action will be performed by the agent even if it's configured. If this option is not selected and device sends empty data, then agent will still perform mapped action (default behavior). 

Reviewer 2: ```suggestion
If the **Ignore empty values** option is selected and a device sends empty data, the agent does not perform any action that is configured. If this option is not selected and a device sends empty data, the agent performs the mapped action (default behavior). 
```
